src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -2483,13 +2485,13 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
         }
 
         // Refrigerant type
-        thisVrfFluidCtrl.RefrigerantName = cAlphaArgs(4);
-        if (Util::FindItemInList(thisVrfFluidCtrl.RefrigerantName, state.dataFluidProps->RefrigData, state.dataFluidProps->NumOfRefrigerants) == 0) {
-            ShowSevereError(state, cCurrentModuleObject + " = " + thisVrfFluidCtrl.Name);
-            ShowContinueError(state, "Illegal " + cAlphaFieldNames(4) + " = " + cAlphaArgs(4));
+        thisVrfFluidCtrl.refrigName = cAlphaArgs(4);
```

Comment:
Grabbing the pointer to the refrigerant here so that we can use the new fluid API.

Reply:
Got it.  And as with the plant component work, the refrig pointer is not owning.  It's just an address, and we need to check null before using it.  Good to go.

Reply:
You don't really need to check it, unless the option of not having a fluid at all is a legitimate option. Pointer ownership is about who has to free the memory when it is no longer used, not about checking for `nullptr`.

Reply:
I just mean check for null once during get input routines in case there is an invalid refrigerant string passed in from user input to the GetRefrig function.  Once state is all setup, definitely shouldn't be checking it over and over again.